tests/ui/README.md

```diff
@@ -0,0 +1,104 @@
+# Integration Tests for the [Mozilla Addons Website][amo].
+## How to run the tests locally
+### Clone the repository
+
+If you have cloned this project already then you can skip this, otherwise you'll
+need to clone this repo using Git. If you do not know how to clone a GitHub
+repository, check out this [help page][git-clone] from GitHub.
+
+If you think you would like to contribute to the tests by writing or maintaining
+them in the future, it would be a good idea to create a fork of this repository
+first, and then clone that. GitHub also has great instructions for
+[forking a repository][git-fork].
+
+### Server Install
+
+Follow the instructions found [here][addons-server-docs].
+
+### Run the tests
+
+Included in the docker-compose file is an image containing firefox nightly. [tox][Tox]
+is our test environment manager and [pytest][pytest] is the test runner.
+
+To run the tests, execute the command below:
+```sh
+docker-compose exec --user root selenium-firefox tox -e ui-tests
+```
+
+### Adding a test
+
+The tests are written in python using a POM, or Page Object Model. The plugin we use for this is called [pypom][pypom]. Please read the documentation there for good examples
+on how to use the Page Object Model when writing tests.
+
+The pytest plugin that we use for running tests has a number of advanced command
+line options available too. The full documentation for the plugin can be found [here][pytest-selenium].
+
+## Additional Information
+
+The tests run against the newest version of the [AMO][amo] frontend using a docker image provided by [addons-frontend][addons-frontend]. You can view the frontend after the build has been completed at ```localhost:4000```.
```

Comment:
We should link to `olympia.test:4000` since that's the official host we are recommending. That should work right?

Reply:
It should yes